test/e2e/framework/apparmor.go

```diff
@@ -23,7 +23,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/kubernetes/pkg/security/apparmor"
-	"k8s.io/kubernetes/test/e2e/framework"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
```

Comment:
This dependency way is not valid (e2e core framework to e2e framework subpackages).
I hesitate to merge this into the core framework at this time.

Reply:
I also don't like this dependency here.
But if we looks into other files in framework, there are many place also has this dependency issue.
We need to sort out the calling relationship of each components of e2e framework.

Reply:
@danielqsj 
> But if we looks into other files in framework, there are many place also has this dependency issue.

Yes, you are right.
Then we are trying to remove such invalid dependencies from the existing places.
I feel it is not necessary to add more.

Reply:
@oomichi does one e2e framework subpackages has dependency of another e2e framework subpackage seems good ?
If it's good, I can move the `apparmor.go` to a dedicated subpackage like `security`.

Reply:
But this will add the dependency of the package `e2e/framework` directly.

Reply:
I opened an issue to track this issue: https://github.com/kubernetes/kubernetes/issues/85193

Reply:
@danielqsj 
> @oomichi does one e2e framework subpackages has dependency of another e2e framework subpackage seems good ?

That is a nice point. IIUC we never discussed the dependency between subpackages.
I guess the dependency could be fine.

Reply:
sure. Re-implemented this change by moving it to a new subpackage.
@oomichi Please help review it ? Thanks